aptos-move/move-examples/tic-tac-toe/Move.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = 'tic-tac-toe'
+version = '1.0.0'
+
+[addresses]
+tic_tac_toe = "_"
+
+[dependencies.AptosFramework]
+git = 'https://github.com/aptos-labs/aptos-core.git'
+rev = 'main'
+subdir = 'aptos-move/framework/aptos-framework'
```

Comment:
Since all other example projects use local dependencies, it might be better we also do it that way.